src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -584,22 +584,35 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
-  zoomOnCurrentAsset = async (currentAsset: Element | null) => {
-    if (!currentAsset || !this.pinchZoomInstance) {
+  handleSearchResult = (foundElement: Element | null) => {
+    this.zoomOnElement(
+      foundElement,
+      this.props.searchZoom || this.props.initialZoom
+    );
+  };
+
+  /**
+   * @param element - any annotation element on SVG, typically asset or document
+   * @param zoomLevel
+   */
+  zoomOnElement = (element: Element | null, zoomLevel?: number) => {
```

Comment:
As we're not calling `zoomOnElement` without `zoomLevel`, but are calling it w/ `props.initialZoom` in several places, perhaps it'd also be best to assign it a default value of `props.initialZoom`?

Reply:
Ok, I'll check if I can apply it.

Reply:
nah, I don't like where it goes... 

```
zoomLevel: number | undefined = this.props.initialZoom
```
still can be undefined, plus I think it's clearer to read `zoomOnElement(foo, this.props.initialZoom)` than `zoomOnElement(foo)` as it is explicit about the level it will be zoomed to.

Reply:
It's still the same type yes. 
I get your point about explicitness, let's leave it as-is then

Reply:
changed my mind